bittide/src/Bittide/Arithmetic/Time.hs

```diff
@@ -71,3 +79,35 @@ trueFor _ clk rst =
   transF counter = \case
     True -> satSucc SatBound counter
     _    -> 0
+
+-- | Rises after the incoming signal has been 'True' for the specified amount of
+-- time. The time is specified in milliseconds. Example invocation:
```

Comment:
The documentation here still can be improved. It took me a bit to realize that the specified time is just `stepSize` * `someLimit`. Why not adding that fact to the documentation as well.

And it should also be noted that this introduces an error if `stepSize` can't  be captured precisely by the selected clock domain, as the formula mentioned above does not take this into account.

Reply:
The documentation was a mess in other ways too. How do you feel about the current state?

Reply:
The current state delivers a more precise description in that sense.